snitun/multiplexer/channel.py

```diff
@@ -84,15 +159,19 @@ async def write(self, data: bytes) -> None:
         )
 
         try:
-            async with asyncio_timeout.timeout(5):
-                await self._output.put(self.id, message)
-        except TimeoutError:
-            _LOGGER.debug("Can't write to peer transport")
-            raise MultiplexerTransportError from None
-
-        if not self._throttling:
-            return
-        await asyncio.sleep(self._throttling)
+            # Try to avoid the timer handle if we can
+            # all to the queue without waiting
```

Comment:
"all" sounds like a typo. It should probably be a verb like add.

Reply:
https://github.com/NabuCasa/snitun/pull/361